client/ayon_core/pipeline/workfile/utils.py

```diff
@@ -513,8 +525,18 @@ def save_next_version(
                 product_type="workfile"
             )
 
+    # Re-use comment if is not set
+    if comment is None:
+        if current_workfile is not None:
+            # Use 'comment' from the current workfile if is set
+            comment = current_workfile.comment
+        elif last_workfile is not None:
+            # Use 'comment' from the last workfile
+            comment = last_workfile.comment
```

Comment:
I wouldn't have expected this myself, anyway - fine with me. Just want to point out that this is new behavior, different from how the Version Up logic previously worked.

Reply:
I actually don't have reason for this. Probably should be removed...